### DISCUSSION TITLE
Any experience on running via Saucelabs?

### DISCUSSION THREAD

**Discussion by cambiph** [leancodepl/patrol#1816] [Q&A]
Hey all,

I was wondering if anybody has any experience with running Patrol tests on Saucelabs.
I checked the docs and tried setting it up as they do for Espresso tests, but then only one test runs.
Any idea if this is something Saucelabs or Patrol related?

Kind regards

**Comment by bartekpacia** [comment]
Hi, nope, we haven't tried. But it looks similar to BrowserStack, and since Patrol works on BS, it should work there as well (maybe after some small modifications - every device farm is a bit different).

cc @jBorkowska @fylyppo
